app/models/setting.rb

```diff
@@ -109,6 +109,7 @@ def defaults
         "process.polls": true,
         "process.budgets": true,
         "process.legislation": true,
+        "process.legislation_proposals": true,
```

Comment:
Hello @javierm , it is because of the `feature_flag :legislation_proposals` line in the controller. The same can be found for proposals, budgets ... But not for comments. 
It will perform an automatic check if the feature is disabled or not, and raise an error if not.
So, should I keepf the `feature_flag :legislation_proposals` in the controller and keep the setting as if ?
Or, should I get rid of the enabled/disabled feature in the controller and remove this line from setting.rb ?

Reply:
Oh, right; I missed the line in the controller :sweat_smile:. I think it would be enough to remove this line and change the controller to use `feature_flag :legislation`, so it's consistent with the rest of the sections related to legislation :thinking:.

Reply:
I have removed that setting.